smoketest/systemlimits_test.go

```diff
@@ -159,31 +204,111 @@ func TestSystemLimits(t *testing.T) {
 				7. set limit to -1
 				8. create (should work)
 			*/
+			t.Parallel()
+			h := harness.NewHarness(t, sql, "limit-configuration")
+			defer h.Close()
+
+			doQL := func(t *testing.T, query string) (string, string) {
+				t.Helper()
+				g := h.GraphQLQuery2(query)
+				if len(g.Errors) > 1 {
+					for _, err := range g.Errors {
+						t.Logf(err.Message)
+					}
+					t.Fatalf("got %d errors; want 0 or 1", len(g.Errors))
+				}
+				if len(g.Errors) == 0 {
+					var m map[string]interface{}
+
+					err := json.Unmarshal(g.Data, &m)
+					if err != nil {
+						t.Fatalf("got err='%s'; want nil", err.Error())
+					}
+					id, _ := getID(m)
+					return id, ""
+				}
+
+				return "", g.Errors[0].Message
+			}
+
+			doQuery := func(t *testing.T, query string) string {
+				id, errMsg := doQL(t, query)
+				assert.Empty(t, errMsg, "error message")
+				return id
+			}
+			doQueryExpectError := func(t *testing.T, query string, expErr string) {
+				_, errMsg := doQL(t, query)
+				t.Log(errMsg)
+				t.Log(expErr)
+				assert.Contains(t, errMsg, expErr, "error message")
+			}
+
+			tmpl := template.New("uuids")
+			tmpl.Funcs(template.FuncMap{
+				"uuid": func(id string) string {
+					if id == "phone" {
+						p := h.Phone("")
+						return p
+					}
+					if uuid, ok := uuids[id]; ok {
+						return uuid
+					}
+					uuid := h.UUID(id)
+					uuids[id] = uuid
+					return uuid
+				},
+			})
```

Comment:
I think `h.UUID()` would be easier to read than going through empty template rendering, or am I missing something?

Reply:
I agree. `h.UUID()` would be much easier, but with the tests running in parallel the harness is now only accessible within either the `t.Run()` or the helper functions (`doQL()`) I am not sure how to send `h.UUID()` from the test cases. Or is there a way to incorporate the harness within `check()`?

Reply:
The harness can be passed as a parameter, much like we're doing with `t` where needed